geoindicators/src/main/groovy/org/orbisgis/RsuIndicators.groovy

```diff
@@ -619,4 +615,164 @@ static IProcess rsuEffectiveTerrainRoughnessHeight() {
 
                 [outputTableName: outputTableName]
             }
+    )}
+
+/** Performs operations on the linear of road within the RSU scale objects. Note that when a road is located at
+ * the boundary of two RSU, it is arbitrarily attributed to the RSU having the lowest ID in order to not
+ * duplicate the corresponding road.
+ *
+ * @param datasource A connexion to a database (H2GIS, PostGIS, ...) where are stored the input Table and in which
+ * the resulting database will be stored
+ * @param rsuTable the name of the input ITable where are stored the RSU geometries
+ * @param roadTable the name of the input ITable where are stored the road geometries
+ * @param operations the name of the geospatial variables to calculated using the linear of road:
+ *      -> "rsu_road_direction_distribution": The direction of each segment of road is calculated. The percentage of linear of road
```

Comment:
reduce to road_direction_distribution because the name of the process already specifies that we are runing a rsu process ?

Reply:
you are right. In this case, it would be the same for all indicator names. Should I add an issue in order to transform all the geoindicator names at the end of the "coding session" ?

Reply:
yep my friend